### ISSUE TITLE
Windows installation script does not install pixi

### ISSUE THREAD

**Issue by traversaro** [prefix-dev/pixi#150]
I am following the instructions in the README to install pixi on Windows, and the command closes the Powersehll windows, on both Powershell and Powershell with Admin Permissions. This is the output before the windows is closed:
~~~                                                                                                                     Writing web request                                                                                                        Writing request stream... (Number of bytes written: 2834112)                                                                                                                                                                                PS C:\WINDOWS\system32> C:\Users\STraversaro\AppData\Local\pixi\bin^C
PS C:\WINDOWS\system32> iwr -useb https://raw.githubusercontent.com/prefix-dev/pixi/main/install/install.ps1 | iex
This script will automatically download and install Pixi (latest) for you.
Getting it from this url: https://github.com/prefix-dev/pixi/releases/latest/download/pixi-x86_64-pc-windows-msvc.zip
The binary will be installed into 'C:\Users\STraversaro\AppData\Local\pixi\bin'
~~~

**Comment by traversaro** [comment]
I manually removed the try/catch, and the error is:
~~~
Expand-Archive : .tmp is not a supported archive file format. .zip is the only supported archive file format.
At C:\Users\STraversaro\test2\install.ps1:35 char:5
+     Expand-Archive -Path $TEMP_FILE -DestinationPath $PIXI_DIR -Force
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (.tmp:String) [Expand-Archive], IOException
    + FullyQualifiedErrorId : NotSupportedArchiveFileExtension,Expand-Archive
~~~

Apparently `[System.IO.Path]::GetTempFileName()` returns a filename that ends with `.tmp`, and `Expand-Archive` does not like this.

**Comment by ruben-arts** [comment]
Thanks for the issue! Fixed it in #152

**Comment by traversaro** [comment]
Thanks for the fix!
